plugins/filter_kubernetes/kube_meta.c

```diff
@@ -1335,6 +1467,7 @@ int flb_kube_meta_get(struct flb_kube *ctx,
                        meta->cache_key, meta->cache_key_len,
                        (void *) &hash_meta_buf, &hash_meta_size);
     if (ret == -1) {
+        flb_plg_info(ctx->ins, "testing: get K8s metadata");
```

Comment:
remove this

Reply:
It is for testing purpose. Will remove it in next commit